src/styles/theme/oc-drop.scss

```diff
@@ -0,0 +1,6 @@
+.oc-drop {
+  width: 250px;
+  border: 1px solid $muted-background;
+  background-color: $background;
```

Comment:
Codacy found an issue: [Expected background-color to come before border (order/properties-alphabetical-order)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5880368)